code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -885,14 +885,17 @@ Pass a positive integer as an argument to override a bot's default speed.
 /mob/living/simple_animal/bot/ui_act(action, params)
 	. = ..()
 	if(.)
-		return
+		return TRUE
```

Comment:
This is implicitly true already, ok if you want to be more clear though

Reply:
yeah i always get confused because some procs use the opposite, so I wanted it to be clear.